Common/ac/gamestructdefines.h

```diff
@@ -225,6 +225,12 @@ struct SpriteInfo
 // multiple lines, and similar cases.
 struct FontInfo
 {
+    enum AutoOutlineStyleT : int
```

Comment:
Please, don't add suffixes like 'T' to the enum names, we never had this naming convention before.